build.sbt

```diff
@@ -23,7 +23,7 @@ import uk.gov.hmrc.SbtAutoBuildPlugin
 
 val appName = "vat-correspondence-details-frontend"
 
-val bootstrapPlayVersion       = "3.4.0"
+val bootstrapPlayVersion       = "5.4.0"
```

Comment:
Could you please also still remove the unused SbtAutoBuilPlugin import and remove dependencies that aren't needed anymore (govuk-template, play-test)?